_layouts/text-home.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+
+    <div class="usa-section">
+        <div class="grid-container">
+            <div class="grid-row grid-gap">
+                <div class="usa-layout-docs__sidenav desktop:grid-col-3">
+                    <nav aria-label="Secondary navigation">
+                        {% include nav.html %}
```

Comment:
I saw you had the nav content in the front matter, but the include has the full HTML with content embedded. Did you need help figuring out how to use a Jekyll template, or is this not what you intended to merge?

Reply:
yeah i need some help on this @pglevy